src/index.js

```diff
@@ -28,15 +29,32 @@ module.exports = {
     start: compileProject,
     watcher: async function ({ filename, /* event, */ inventory }) {
       if (filename.endsWith('.ts') || filename.endsWith('.tsx')) {
-        // Second pass filter by Lambda dir
+        let { cwd } = inventory.inv._project
+        let globalTsConfig = getTsConfig(cwd)
+        let tsCompilerOptions = require(globalTsConfig).compilerOptions
+        if (tsCompilerOptions) {
+          let recompileProject = false
+          let tsPaths = tsCompilerOptions.paths || {}
+          for (const [_, paths] of Object.entries(tsPaths)) {
+            paths.map((p) => {
+              if (filename.startsWith(resolve(cwd, tsCompilerOptions.baseUrl, p).replace(/\/?\*$/, ""))) {
```

Comment:
I think there may be a bug here: what about folks who use Windows paths? We need to account for both slashes and backslashes.

This can probably be broken into another line or two to make it a bit more readable.